pre_commit_hooks/mixed_line_ending.py

```diff
@@ -0,0 +1,250 @@
+import argparse
+import logging
+import os
+import re
+import sys
+
+from enum import Enum
```

Comment:
Using [requirements-tools's visualize-requirements](https://github.com/Yelp/requirements-tools) (that I wrote as well, lol), it seems that `enum34` is being pulled in *implicitly* from flake8:

```
# first comment out `-e .`
$ visualize-requirements requirements-dev.txt
 coverage
 flake8
   - pyflakes<1.6.0,>=1.5.0
   - enum34
   - configparser
   - mccabe<0.7.0,>=0.6.0
   - pycodestyle>=2.0.0,<2.4.0
 mock
   - six>=1.9
   - funcsigs>=1
   - pbr>=0.11
 pre-commit
   - aspy.yaml
     - pyyaml
   - six
   - cached-property
   - pyyaml
   - identify>=1.0.0
   - virtualenv
   - nodeenv>=0.11.1
 pytest
   - setuptools
   - py>=1.4.33
```

The `extras_require` syntax above just tells (recent enough) pip that "for python2.7, I have these dependencies" -- pip will know what to do automatically.